486_Project/Characters_Class.py

```diff
@@ -97,23 +164,252 @@ def __init__(self):
         self.name = "Wolf"
 
         # [strength, dexterity, intelligence, constitution]
-        self.stats = [3, 6, 3, 5]
+        self.stats = [4, 6, 4, 5]
 
         # armor for this enemy
-        self.wolf_armor = 1
+        self.enemy_armor = 1
 
         # health points
-        self.wolf_max_hp = self.stats[3] * 4
-        self.wolf_hp = self.wolf_max_hp
+        self.enemy_max_hp = self.stats[3] * 4
+        self.enemy_hp = self.enemy_max_hp
 
         # enemy menu for choices during combat
-        self.wolf_menu = ["attack", "defend", "skills"]
+        self.enemy_menu = ["attack", "defend", "skills"]
 
         # enemy skills to use during combat
-        self.wolf_skills = ["maul", "rend"]
+        self.enemy_skills = ["maul", "rend"]
```

Comment:
Non-attacking skills might be an interesting addition here! For example, one thing wolves might do would be "lick wounds", which might heal them slightly.